pkg/config/config.go

```diff
@@ -55,6 +55,7 @@ var DefaultConfig = Config{
 	Metrics:               metrics.DefaultConfig,
 	Integrations:          DefaultVersionedIntegrations,
 	EnableConfigEndpoints: false,
+	EnableUsageReport:     false,
```

Comment:
Since we're changing the default
```suggestion
	EnableUsageReport:     true,
```